src/challenges/challengeOne/1.Accordion/Accordion.test.tsx

```diff
@@ -1,8 +1,30 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Accordion } from "./Accordion";
-
-// describe("Accordion", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Accordion } from "./Accordion";
+
+describe("Accordion", () => {
+  // const onClickMock = jest.fn();
+  const testHeaderText = "Accordion title";
+  const testChildren = <p>Accordion content</p>;
+
+  const renderComponent = () => {
+    const defaultProps = {
+      title: testHeaderText,
```

Comment:
You can just add the header text directly, the abstraction doesn't do much.

```suggestion
      title "Accordion title",
```